scripts/convert_llama_to_dcp.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+import json
+from pathlib import Path
+
+import torch
+import torch.distributed.checkpoint as DCP
+
+@torch.inference_mode()
+def convert_llama_weights(input_dir, output_dir):
+    with open(args.input_dir / "params.json", "r") as f:
+        params = json.load(f)
+    n_layers = params["n_layers"]
+    n_heads = params["n_heads"]
+    dim = params["dim"]
+    dims_per_head = dim // n_heads
+
+    checkpoint_list = sorted([file for file in input_dir.rglob("*.pth")])
+    print(f"Loading from these files: {checkpoint_list}.")
```

Comment:
Let's use logging instead of printing, similar to https://github.com/pytorch/torchtitan/blob/main/train.py#L51

Reply:
When you replace print with logging, could we say "Loading original Llama weights from ..."?